airqo-platform/AirQo-api
# [ci/cd] get changed microservices

# detect changed microservices in a mono-repo GitHub structure

**_WHAT DOES THIS PR DO?_**
This PR aims at tracking changes in microservices for the purpose of deployment. It prints the branch name and list of changed folders 
**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-330]
**_WHAT IS THE LINK TO THE PR BRANCH_**
N/A
**_HOW DO I TEST OUT THIS PR?_**
check out this branch and `cd` to `AirQo-api/pipeline`. Run `./deploy-changed.sh` on your `cmd` or `terminal`
**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
N/A
**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A
**_ARE THERE ANY RELATED PRs?_**
N/A



[PLAT-330]: https://airqoteam.atlassian.net/browse/PLAT-330
